test/units/EverlongPortfolio.t.sol

```diff
@@ -253,7 +253,7 @@ contract TestEverlongPositions is EverlongTest {
         assertApproxEqAbs(
             IERC20(everlong.asset()).balanceOf(address(everlong)),
             everlong.targetIdleLiquidity(),
-            1e18
+            everlong.targetIdleLiquidity() / 10
```

Comment:
This is a big tolerance. We should add a `TODO` or `FIXME` if you're not planning to fix this in this PR.

Reply:
Totally agree, added a TODO.